README.md

```diff
@@ -1,24 +1,40 @@
 # UZHSWAP Contracts (UniswapV3 Fork)
 
-This Hardhat project can be used to deploy all necessary contracts to the [UZH Ethereum network](http://uzheth.business.uzh.ch)
+This Hardhat project can be used to deploy all necessary contracts to
+the [UZH Ethereum network](http://uzheth.business.uzh.ch)
+
+## Setup
+
+In the root folder create a `.env` file with the following values:
+
+    UZH_PRIVATE_KEY="<private key of a valid UZHETH account to deploy contracts>"  
+    GANACHE_PRIVATE_KEY="<private key of a valid GANACHE account to deploy contracts>"  
+    GANACHE_URL="http://127.0.0.1:7545"
+    UZH_URL="http://130.60.244.246:8545"  
```

Comment:
http://uzheth.business.uzh.ch

Reply:
this link was not working when I tried it on my side :D
should I change it anyway?

Reply:
Probably not  then 👀